README.md

```diff
@@ -5,7 +5,32 @@
 The Zigbee R23 add-on for the nRF Connect SDK provides support for developing Zigbee applications based on the third-party precompiled ZBOSS stack. This stack is included as the ZBOSS Zigbee R23 stack library. In combination with the nRF Connect SDK, the Zigbee R23 add-on allows for development of low-power connected solutions.
 
 ## Getting started
-To get started with Zigbee R23 add-on for nRF Connect SDK, follow [documentation](https://docs.nordicsemi.com/bundle?cluster=true&exclude_metadata_filter.field=display-type&exclude_metadata_filter.value=inline&labelkey=addon-zigbee-r23).
+To get started with Zigbee R23 add-on for nRF Connect SDK, follow [documentation](https://nrfconnect.github.io/ncs-zigbee/).
+
+## Documentation
```

Comment:
I think we've discussed it some time ago @milewr , but I don't think the documentation build is necessary, given the documentation is pre-built.